34th-Sopt-Assignment/34th-Sopt-Assignment/Main/views/CarouselViewController.swift

```diff
@@ -0,0 +1,155 @@
+            (UIScreen.main.bounds.width - Self.itemSize.width) / 2.0
+        }
+        static var collectionViewContentInset: UIEdgeInsets {
+            UIEdgeInsets(top: 0, left: Self.insetX, bottom: 0, right: Self.insetX)
+        }
+    }
+    
+    private let collectionViewFlowLayout = UICollectionViewFlowLayout()
+    
+    private lazy var collectionView: UICollectionView = {
+        let view = UICollectionView(frame: .zero, collectionViewLayout: self.collectionViewFlowLayout)
+        return view
+    }()
+    
+    private let indicatorStackView = UIStackView()
+    
+    private var items = MainposterModel.dummy()
+    private var previousIndex: Int?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setStyle()
+        setDelegate()
+        setLayout()
+        
+    }
+    
+    func setLayout() {
+        [indicatorStackView].forEach { [weak self] view in
+            guard let self = self else { return }
+            self.view.addSubview(view)
+        }
+        
+        view.addSubview(collectionView)
+        collectionView.snp.makeConstraints {
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(498)
+            $0.top.equalToSuperview()
+            
+        }
+        
+        view.addSubview(indicatorStackView)
+        indicatorStackView.snp.makeConstraints {
+            $0.leading.equalToSuperview().inset(15)
+            $0.top.equalTo(collectionView.snp.bottom).offset(14)
+        }
+        
+    }
+    
+    func setStyle() {
+        collectionViewFlowLayout.do {
+            $0.scrollDirection = .horizontal
+            $0.itemSize = Const.itemSize
+            $0.minimumLineSpacing = Const.itemSpacing
+            $0.minimumInteritemSpacing = 0
+        }
+        collectionView.do {
+            $0.isScrollEnabled = true
+            $0.showsHorizontalScrollIndicator = false
+            $0.showsVerticalScrollIndicator = true
+            $0.backgroundColor = .clear
+            $0.clipsToBounds = true
+            $0.register(MainPosterCollectionViewCell.self, forCellWithReuseIdentifier: MainPosterCollectionViewCell.id)
+            $0.isPagingEnabled = false
+            $0.contentInsetAdjustmentBehavior = .never
+            $0.contentInset = Const.collectionViewContentInset
+            $0.decelerationRate = .fast
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        for _ in self.items {
+            let indicatorView = UIView()
+            indicatorView.backgroundColor = .gray
+            indicatorView.layer.cornerRadius = 3
+            indicatorView.translatesAutoresizingMaskIntoConstraints = false
+            self.indicatorStackView.addArrangedSubview(indicatorView)
+            NSLayoutConstraint.activate([
+                indicatorView.widthAnchor.constraint(equalToConstant: 5),
+                indicatorView.heightAnchor.constraint(equalToConstant: 5),
+            ])
```

Comment:
저희가 import SnapKit을 해줬기 때문에 
```suggestion
     indicatorView.snp.makeConstraints { make in
            make.width.height.equalTo(5)
        }
```
이런식으로 제약 방식을 통일 시켜주는 것도 좋을 것 같습니다!